commit 682014bb9469090f00b4614cd21e8e3cc47637a1
Author: Arnout Engelen <arnout@bzzt.net>
Date:   Thu Oct 2 13:10:21 2025 +0200

    chore: add PEP-517 metadata to specify Poetry as build tool

diff --git a/backend/pyproject.toml b/backend/pyproject.toml
index 8bfb28a1..4ebc7aea 100644
--- a/backend/pyproject.toml
+++ b/backend/pyproject.toml
@@ -6,6 +6,7 @@ license = "BSD-3-Clause"
 authors = [
     "Stefan Fleckenstein <stefan.fleckenstein@maibornwolff.de>",
 ]
+package-mode = false
 
 [tool.poetry.dependencies]
 python = ">= 3.10, < 3.14"
@@ -99,3 +100,7 @@ code_quality = ["flake8", "flake8-isort", "black", "pylint", "pylint-django", "p
 [tool.black]
 line-length = 120
 target-version = ['py312']
+
+[build-system]
+requires = ["poetry-core>=1.0.0"]
+build-backend = "poetry.core.masonry.api"
